### DISCUSSION TITLE
Authentication Backend Support

### DISCUSSION THREAD

**Discussion by USAFrenzy** [activexray/Doplarr#119] [Q&A] [answered]
I haven't found anything regarding authentication backend support with Doplarr, however, I use Authelia with all my containers and hate the fact that I had to remove the authelia backend from Sonarr and Radarr in order to integrate them with Doplarr. With authelia active, Doplarr would just spin and spin without sending the queries to either Sonarr or Radarr.

Have I missed a known work-around or is this out of scope of the project?

**Comment by activexray** [comment, marked as answer]
So, your radarr and sonarr are public-facing and that's why you need a separate auth frontend? Looking around, it seems people usually have bypasses for API endpoints for this purpose (https://github.com/Just5KY/SkyPi/blob/main/Authelia/README.md) considering your API key is the secret, there's no point having a second auth layer.

**Comment by USAFrenzy** [comment]
That actually makes a lot of sense! Also, I appreciate you providing a link for a supplemental option - I probably should have made my way on over there for a look before opening up a thread here, so I apologize for that. Thank you again.
